[2024-11-02T13:30:32.959+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-11-02T13:30:32.979+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-11-02T13:30:32.980+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-11-02T13:30:33.009+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotSchemaSubmitOperator): submit_schemas> on 2024-04-01 00:00:00+00:00
[2024-11-02T13:30:33.016+0000] {standard_task_runner.py:57} INFO - Started process 3391 to run task
[2024-11-02T13:30:33.021+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'schema_dag', 'submit_schemas', 'scheduled__2024-04-01T00:00:00+00:00', '--job-id', '1279', '--raw', '--subdir', 'DAGS_FOLDER/schema_dag.py', '--cfg-path', '/tmp/tmppk5d7u_t']
[2024-11-02T13:30:33.026+0000] {standard_task_runner.py:85} INFO - Job 1279: Subtask submit_schemas
[2024-11-02T13:30:33.097+0000] {task_command.py:415} INFO - Running <TaskInstance: schema_dag.submit_schemas scheduled__2024-04-01T00:00:00+00:00 [running]> on host 5e812a6a7c02
[2024-11-02T13:30:33.225+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='minhtranquang' AIRFLOW_CTX_DAG_ID='schema_dag' AIRFLOW_CTX_TASK_ID='submit_schemas' AIRFLOW_CTX_EXECUTION_DATE='2024-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-01T00:00:00+00:00'
[2024-11-02T13:30:33.226+0000] {pinot_schema_operator.py:33} ERROR - An error occured: 'function' object has no attribute 'glob'
[2024-11-02T13:30:33.243+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=schema_dag, task_id=submit_schemas, execution_date=20240401T000000, start_date=20241102T133032, end_date=20241102T133033
[2024-11-02T13:30:33.274+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-02T13:30:33.321+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
